calico-enterprise_versioned_docs/version-3.19-1/_includes/components/ReqsSys.js

```diff
@@ -33,6 +33,8 @@ function NodeRequirementsEnt(props) {
                 <li>Ubuntu 20.04 and 22.04</li>
                 <li>RHEL 8 and 9</li>
                 <li>Debian 10</li>
+                <li>Azure Linux Container Host</li>
+                <li>Rocky Linux</li>
```

Comment:
Version numbers?

Reply:
Don't think Azure Linux container host specifies version numbers when installing. This is how it's installed https://learn.microsoft.com/en-us/azure/azure-linux/quickstart-azure-cli. It doesnt use a version number to install like how we do for Ubuntu or RHEL. More on that from AKS doc: 

`Azure Linux has a focus on stability, often backporting fixes in core components like the kernel or openssl. It also limits substantial changes or significant version bumps to major release boundaries (for example, Azure Linux 2.0 to 3.0), which prevents customer outages.`

Reply:
Yeah, I had a look myself and it's not easy to get a sense of what's going on. Let's go with this for now, and we can adjust later if we need.